helm/fluent-logshipping-app/templates/configmap.yaml

```diff
@@ -101,6 +101,11 @@ data:
         Skip_Long_Lines   ${input_skip_long_lines}
 
   filters.conf: |
+    {{- if .Values.additional_filters }}
```

Comment:
Sorry to be pedantic but would extra_filters not be best here?

Also, why use underscores when we use camel cases in other places like `Values.outputs.elasticsearch.inputLogTypes`

Reply:
Sure, I'll update... It's better for consistency